vayesta/ewf/fragment.py

```diff
@@ -306,7 +306,9 @@ def get_solver_options(self, solver):
         has_actspace = (
             (solver == "TCCSD")
             or ("CCSDt'" in solver)
+            or ("CCSDt" in solver)
             or (solver.upper() == "EBCC" and self.opts.solver_options["ansatz"] == "CCSDt'")
+            or (solver.upper() == "EBCC" and self.opts.solver_options["ansatz"] == "CCSDt")
```

Comment:
better one line,  `...self.opts.solver_options["ansatz"] in {"CCSDt", "CCSDt'"}`